content/020_event_engine_setup/26_config_workshop_req.ee.md

```diff
@@ -56,4 +56,29 @@ the EKS IAM authentication, so we will disable it and rely on the IAM role inste
    If the IAM role is not valid, <span style="color: red;">**DO NOT PROCEED**</span>. Go back and confirm the steps on this page.
    {{% /notice %}}
 
+
+1. Copy and run (paste with **Ctrl+P** or **CMD+P**) the commands below.
```

Comment:
Can you throw this under content/030_self_guided_setup/36_config_workshop_req.md? the EE track will have all of this bootstrapped to the Cloud9 environment.

Reply:
The kubectl setup appears to already be in the self guided setup.  Am I missing something?

Reply:
This PR has it added to the ee.md file which is specifically for the event engine track